src/test/java/org/springframework/samples/petclinic/rest/controller/PetRestControllerTests.java

```diff
@@ -219,4 +221,46 @@ void testDeletePetError() throws Exception {
             .andExpect(status().isNotFound());
     }
 
+    @Test
```

Comment:
In general, please make sure that all endpoints mentioned in task are covered with test cases.